code/game/objects/effects/decals/turfdecal/vgdecals.dm

```diff
@@ -0,0 +1,119 @@
+// Full credit goes to VG station for these assets. https://github.com/vgstation-coders/vgstation13
+// All items in this .dm and the associated .dmi were made by VG station and all credit should go to them.
+
+// -<| IMPORTANT MAPPER NOTE |>-
+// Change the 'color' variable on any white sprite to simply recolour it!
```

Comment:
Don't encourage mappers to varedit, tell them to make a subtype of what they want recoloured.